packages/core/src/js/utils/base64ToUint8Array.ts

```diff
@@ -0,0 +1,29 @@
+/* eslint-disable no-bitwise */
+export const base64ToUint8Array = (base64?: string): Uint8Array | undefined => {
+  if (!base64) return undefined;
+
+  const cleanedBase64 = base64.replace(/^data:.*;base64,/, ''); // Remove any prefix before the base64 string
+  const chars = 'ABCDEFGHIJKLMNOPQRSTUVWXYZabcdefghijklmnopqrstuvwxyz0123456789+/';
+  const bytes: number[] = [];
+
+  let buffer = 0;
+  let bits = 0;
+
+  for (const char of cleanedBase64) {
+    if (char === '=') break;
+
+    const value = chars.indexOf(char); // Validate each character
+    if (value === -1) return undefined;
+
+    buffer = (buffer << 6) | value; // Shift 6 bits to the left and add the value
+    bits += 6;
+
+    if (bits >= 8) {
+      // Add a byte when we have 8 or more bits
+      bits -= 8;
+      bytes.push((buffer >> bits) & 0xff);
+    }
+  }
+
+  return new Uint8Array(bytes);
+};
```

Comment:
This is quite resource intensive, we do some of these conversion in the JS already, but it should be avoided as much as possible.

---

I this case I would leave that up to the users even thought it makes it a bit harder to use the API (depending on how they get the screenshot)

But it enables user to use conversion lib that are implemented in cpp with much better performance.